Node running as a consumer panics when stopped

Steps to reproduce:

1. Follow this [tutorial](https://gist.github.com/anjmao/e084e254d8240bbd65016a7a5b50ec85) to connect as a consumer.
2. Stop consumer using cli command `stop`.

```
2019-10-09T07:39:51.579683 [Error] [cmd/commands/daemon] terminating application due to error: accept tcp 127.0.0.1:4050: use of closed network connection

2019-10-09T07:39:51.581356 [Info] [dvpn-web-server] server exited
2019-10-09T07:39:51.581413 [Info] [core/node] web server stopped
2019-10-09T07:39:51.581462 [Info] [core/node] NAT pinger stopped
2019-10-09T07:39:51.581807 [Info] [NATPinger] stop pinger called
2019-10-09T07:39:51.581959 [Info] [Noop firewall]Rules reset was requested
2019-10-09T07:39:51.58222 [Info] Good bye
2019-10-09T07:39:51.588818 [Info] [nat] NAT rules cleared
2019-10-09T07:39:51.596187 [Info] [nat] IP forwarding disabled
2019-10-09T07:39:51.596224 [Info] [core/node] no active connection - proceeding
2019-10-09T07:39:51.596241 [Info] [core/node] API stopped
panic: send on closed channel

goroutine 1 [running]:
github.com/mysteriumnetwork/node/vendor/github.com/koron/go-ssdp.(*Advertiser).Bye(0xc00021d030, 0x0, 0xc00062b7b0)
	/Users/anjmao/go/src/github.com/mysteriumnetwork/node/vendor/github.com/koron/go-ssdp/advertise.go:176 +0x11c
github.com/mysteriumnetwork/node/ui/discovery.(*ssdpServer).Stop(0xc000194500, 0x0, 0x0)
	/Users/anjmao/go/src/github.com/mysteriumnetwork/node/ui/discovery/ssdp.go:117 +0x62
github.com/mysteriumnetwork/node/ui/discovery.(*multiDiscovery).Stop(0xc000284f90, 0x52b5228, 0xc000931680)
	/Users/anjmao/go/src/github.com/mysteriumnetwork/node/ui/discovery/discovery.go:55 +0x4a
github.com/mysteriumnetwork/node/ui.(*Server).Stop(0xc00000e440)
	/Users/anjmao/go/src/github.com/mysteriumnetwork/node/ui/server.go:115 +0x49
github.com/mysteriumnetwork/node/core/node.(*Node).Kill(0xc000695090, 0x543dead, 0x0)
	/Users/anjmao/go/src/github.com/mysteriumnetwork/node/core/node/node.go:115 +0x3b8
github.com/mysteriumnetwork/node/cmd.(*Dependencies).Shutdown(0xc0001242c0, 0x0, 0x0)
	/Users/anjmao/go/src/github.com/mysteriumnetwork/node/cmd/di.go:397 +0x21d
github.com/mysteriumnetwork/node/cmd/commands/daemon.NewCommand.func2(0xc00012c160, 0x402fe31, 0x52b4500)
	/Users/anjmao/go/src/github.com/mysteriumnetwork/node/cmd/commands/daemon/command.go:56 +0x2a
github.com/mysteriumnetwork/node/vendor/gopkg.in/urfave/cli%2ev1.Command.Run.func1(0xc00062bb60, 0xc00012c160, 0xc00062bc90)
	/Users/anjmao/go/src/github.com/mysteriumnetwork/node/vendor/gopkg.in/urfave/cli.v1/command.go:185 +0x3b
github.com/mysteriumnetwork/node/vendor/gopkg.in/urfave/cli%2ev1.Command.Run(0x4d3f9ef, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0x4d63eae, 0x22, 0x0, ...)
	/Users/anjmao/go/src/github.com/mysteriumnetwork/node/vendor/gopkg.in/urfave/cli.v1/command.go:215 +0x9b1
github.com/mysteriumnetwork/node/vendor/gopkg.in/urfave/cli%2ev1.(*App).Run(0xc0000cc000, 0xc0000d6000, 0x5, 0x5, 0x0, 0x0)
	/Users/anjmao/go/src/github.com/mysteriumnetwork/node/vendor/gopkg.in/urfave/cli.v1/app.go:255 +0x6ab
main.main()
	/Users/anjmao/go/src/github.com/mysteriumnetwork/node/cmd/mysterium_node/mysterium_node.go:56 +0x20c
```

--- a/ui/discovery/ssdp.go
+++ b/ui/discovery/ssdp.go
@@ -107,18 +107,18 @@ func (ss *ssdpServer) Start() (err error) {
 }
 
 func (ss *ssdpServer) Stop() error {
+	var closeError error
 	ss.once.Do(func() {
 		close(ss.quit)
+		if ss.ssdp != nil {
+			if err := ss.ssdp.Bye(); err != nil {
+				log.Error().Err(err).Msg("Failed to send SSDP bye message")
+			}
+			closeError = errors.Wrap(ss.ssdp.Close(), "failed to send SSDP bye message")
+		}
 	})
 
-	if ss.ssdp != nil {
-		if err := ss.ssdp.Bye(); err != nil {
-			log.Error().Err(err).Msg("Failed to send SSDP bye message")
-		}
-		return errors.Wrap(ss.ssdp.Close(), "failed to send SSDP bye message")
-	}
-
-	return nil
+	return closeError
 }
 
 func (ss *ssdpServer) serveDeviceDescriptionDocument() (url.URL, error) {

